taxi/forms.py

```diff
@@ -0,0 +1,33 @@
+import re
+
+from django import forms
+from taxi.models import Driver
+
+
+class DriverCreateForm(forms.ModelForm):
+    first_name = forms.CharField(
+        widget=forms.TextInput(attrs={"placeholder": "First Name"})
+    )
+    last_name = forms.CharField(
+        widget=forms.TextInput(attrs={"placeholder": "Last Name"})
+    )
+
+    class Meta:
+        model = Driver
+        fields = ["username"]
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ["license_number"]
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if not re.match(r"^[A-Z]{3}\d{5}$", license_number):
```

Comment:
It's ok to use regular expressions in such cases, but probably in this case to simplify it (for other developers) you can just use two conditions
Or you can add docstring to this method